### PR TITLE
caris reflection

### PR DESCRIPTION
**Pull request by dowdcar**
@kdykman2 @KeshiaLynn13 please review :)

### CODE REVIEW

**Comment by KeshiaLynn13** [comment]
Hi Carrissa, 

I appreciate the clear and succinct connections you were able to make between your four mini reflections and the way you opened with an overview of each (I did something similar in mine). I think your connections between the four pieces make sense, especially in the ways that you note the digital and human as overlapping landscapes explored in all these pieces. However, I am a little confused in that I thought our Reflection 2 was supposed to be a composite of our mini-reflections leading up to this, as opposed to a reflection of the four pieces as a whole. I actually had quite the challenge connecting my four mini reflections and think I would've done a little better had I gone with your method of looking at the pieces as whole.

File: r2.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
...d how people of color, particularly black women, were systematically categorized as non-human in digital search infrastructure because of the corporate logic of neglecting of racism for profit. When searching for "black girls," instead of sources about student organizations or community activism popping up as they do now, pornographic and racist sites would appear instead. This is a part of the slow violence against the black community where black bodies are seen as non-human and disposable as objects by hegemonic white supremacy, leading to current issues like police brutality today. Such instances of sexualization, objectification, and dehumanization are a part of structures of misogynoir and racism in western society, and digital infrastructure developers like Google are slow to change this until they get caught. Like the Gulf War conflict, control of the narrative is taken by the aggressors, being the United States or Big Tech in this example, or more broadly Western Imperialism or White Supremacy. People of color are expected to be in a submissive position compared to white people, and this domination is enacted through physical brutality or the control of digital information.
```
Comment (kdykman2) [inline r2.txt:3]: This slowness to accept/acknowledge very real problems within society is a really large problem on a general scale (popular) too, which is interesting

File: r2.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
...he Gulf War was led by the United States for control over oil in Kuwait, and has resulted in a war-torn radiological landscape of death and continuing conflict for Western imperialistic domination. Digital infrastructures are created and maintained by dominant societal groups, usually straight white men, who seek to promote their views of white masculine supremacy, doing so by controlling digital landscapes and public access to information (and only having racist or sexist results appear for non-dominant groups). Technology used to win WWI and WWII were co-opted by civilians to claim space and reimagine their wartime service in a new light, creating a new worldwide cultural anti-war movement and recreating the audio and music industry as a whole. Lastly, humanity and technology are co-evolving alongside one another, parts of a whole that are tumbling through time along side one another, leading to new digital/human landscapes that are a part of a software/hardware/wetware being. As such, while these articles seem very different from first glance, they have many thematic similarities which are more easily defined through extensive reading in the fields of digital and eco-humanities.
```
Comment (kdykman2) [inline r2.txt:7]: yes- it's definitely interesting to consider Nixon and Noble's articles as almost a continuation of the conversations that Kittler and Winthrop-Young discuss- while the latter warn of the dangers of separating out technology from the human, Nixon and Noble represent the very real problems that have already arisen from just such a separation.

File: r2.txt (hunk 1 above)
Comment (dowdcar) [reply r2.txt:3]: Thank you for verbalizing that comment for me. I was dancing around that point for much of the paper!

File: r2.txt (hunk 2 above)
Comment (dowdcar) [reply r2.txt:7]: Again, I thank you for your eloquence. I added a section about separation of technology and humanity being a disregard of our shared history, as well as digital imperalism

**Pull request merged**